docs/start/smeshing/start.md

```diff
@@ -0,0 +1,79 @@
+
+Visit the [System Requirements](./requirements.md) page to see if your system satisfies the requirements. If it does, congratulations! Continue reading to start smeshing.
+
+## Network Configuration
+
+While setting up a full or smeshing node is as simple as running the Smapp and going through the steps, if you run into any network-related issues, visit the [Network Configuration](./netconfig.md) page. It is also recommended to go through the Network Configuration page to see that all the network-related settings are in order so you can have a smooth, uninterrupted smeshing experience.
+
+If you come across any network-related issue and cannot find its solution on the Network Configuration page, post your problem on Discord (invite given above) and someone should reach out to you with a solution.
+
+## Choosing Your Guide
+
+Now that your system satisfies the requirements, it is time to start smeshing! At this stage, you have two ways to start smeshing:
+
+- If you are new to smeshing and/or do not want to use a Command Line Interface (CLI), you can follow the [Basic Smeshing Guide](./smeshing_basic/install.md).
+- If you are an advanced user and/or want more control over your smeshing, you can follow the [Advanced Smeshing Guide](./smeshing_adv/setup.md).
+
+### Smapp - Basic Smeshing
+
+The simplest way to run a Spacemesh node is by using the [Spacemesh App (Smapp)](https://github.com/spacemeshos/smapp/). You can simply download and install the [latest release](https://github.com/spacemeshos/smapp/releases) for your platform and be up and running in no time. Smapp will automatically download the correct configuration file, and will show you visually the network status, its synchronization status, and the status of PoS initialization and smeshing. Smapp bundles a full `go-spacemesh` instance under the hood and manages it for you, including upgrades. See [our Basic Smeshing guide](./smeshing_basic/install.md) for more information on setting up Smapp and smeshing using it.
+
+Smapp also lets you customize your config file and use the node API directly. Running Smapp does not provide any greater or lesser functionality than running go-spacemesh directly, but some technical users who prefer working with a CLI rather than a GUI may wish to run `go-spacemesh` as a system service, etc. Users running without a GUI will of course have no choice but to run `go-spacemesh` directly via CLI.
+
+### CLI - Advanced Smeshing
+
+You can also run a full or smeshing node entirely from the command line using [go-spacemesh](https://github.com/spacemeshos/go-spacemesh). In addition, the [`postcli`](https://github.com/spacemeshos/post/blob/develop/cmd/postcli/README.md) tool can be used to generate PoS data with much more fine-grain control.
+
+Some advantages of smeshing using `go-spacemesh` and `postcli` over Smapp include:
+
+- `go-spacemesh`:
+    - More lightweight compared to Smapp.
```

Comment:
I think it's false :) Smapp uses go-sm behind the scenes.

Reply:
Isn't running an electronJS + go-sm app relatively more resource-consuming vs. just running go-sm?